opencti-platform/opencti-graphql/src/database/stix-2-0-converter.ts

```diff
@@ -201,3 +205,34 @@ export const convertOpinionToStix = (instance: StoreEntity): SDO.StixOpinion =>
     object_refs: convertObjectReferences(instance),
   };
 };
+
+// CONVERTERS
+export type ConvertFn<T extends StoreEntity, Z extends S.StixObject> = (instance: T) => Z;
+const stixDomainConverters = new Map<string, ConvertFn<any, any>>();
+// TODO add registerConverters for module converters
+
+export const convertToStix_2_0 = (instance: StoreCommon): S.StixObject => {
```

Comment:
in the 2.1 converter you have 2 functions and one that does the final cleanup.
I think behavior must be align

Reply:
for now, the only usage of `convertToStix_2_0` is in `convertStoreToStix` (wich is the common converter and does the cleaning and check)